plugins/search.cpp

```diff
@@ -1618,6 +1644,113 @@ IMPLEMENT_HOOKS(df::viewscreen_joblistst, joblist_search);
 
 
 //
+// START: Look menu search
+//
+
+typedef search_generic<df::viewscreen_dwarfmodest, df::ui_look_list::T_items*> look_menu_search_base;
+class look_menu_search : public look_menu_search_base
+{
+    typedef df::ui_look_list::T_items::T_type elt_type;
+public:
+    bool can_init(df::viewscreen_dwarfmodest *screen)
+    {
+        if (ui->main.mode == df::ui_sidebar_mode::LookAround)
+        {
+            return look_menu_search_base::can_init(screen);
+        }
+
```

Comment:
It would require another hook (this one hooks into `df::viewscreen_dwarfmodest`), but it would probably work. I don't think the search plugin implements anything in adventure mode currently, though, so it's possible that there are other issues.